lib/standard/string.nit

```diff
@@ -2245,13 +2245,23 @@ redef class Char
 end
 
 redef class Collection[E]
-	# Concatenate elements.
+	# String representation of the content of the collection.
+	#
+	# The standard representation is the list of elements separated with comas.
+	#
+	# ~~~
+	# assert [1,2,3].to_s == "[1,2,3]"
+	# assert [1..3].to_s  == "[1,2,3]"
+	# assert (new Array[Int]).to_s == "[]" # empty collection
+	# ~~~
+	#
+	# Subclasses may return a more specific string representation.
 	redef fun to_s
 	do
-		return plain_to_s
+		return "[" + join(",") + "]"
```

Comment:
the interpreter caused some GC issues in some tests with superstring.
Once @R4PaSs job is completed, we will chose the most efficient